Schema/V1/arkivmelding.xsd

```diff
@@ -291,13 +289,13 @@
     </xs:complexType>
 
     <xs:complexType name="avskrivning">
-        <xs:complexContent>
-            <xs:extension base="arkivstruktur:avskrivning">
-                <xs:sequence>
-                    <xs:element name="referanseAvskriverJournalpost" type="n5mdk:referanseAvskriverJournalpost" minOccurs="0"/>
-                </xs:sequence>
-            </xs:extension>
-        </xs:complexContent>
+        <xs:sequence>
+            <xs:element name="avskrivningsdato" type="n5mdk:avskrivningsdato"/>
+            <xs:element name="avskrevetAv" type="n5mdk:avskrevetAv"/>
+            <xs:element name="avskrivningsmaate" type="n5mdk:avskrivningsmaate"/>
+            <xs:element name="referanseAvskrivesAvJournalpost" type="n5mdk:referanseAvskrivesAvJournalpost" minOccurs="0"/>
```

Comment:
Typen må kunne være vilkårlig journalpostreferanse i arkivmeldingen på linje med andre steder der det er journalpostreferanse.

Reply:
Hvis det er greit for deg så tar jeg dette i en egen PR? Ønsker å tydeligere vise at dette er en egen endring. 
Slik det står nå så er det som det var før vi tok bort arkivstruktur tror jeg. 
Men hva tenker du egentlig at typen skal være? Mener du at det skal være likt som i f.eks. referanser brukt i  **journalpostHent**?

Reply:
Ja